app/features/conference/components/Conference.js

```diff
@@ -208,6 +207,17 @@ class Conference extends Component<Props, *> {
      * @returns {void}
      */
     _onVideoConferenceJoined(conferenceInfo: Object) {
+        // Inject some style into the iframe so everything except the filmstrip,
+        // buttons or any input is draggable.
+        const iframe = this._api.getIFrame();
+        const injectStyle = document.createElement('style');
+
+        injectStyle.type = 'text/css';
```

Comment:
Isn't it going to be better if you put this code into separate function?

I think this is good enough for temporary solution but isn't it going to be better if we implement more generic way to inject styles into meet from the iframe api?

Reply:
I can move to a function if that feels better.

As for having an API, I don’t disagree, but that isn’t orthogonal to this. We can use that, once / if we have it.

Reply:
sure!